docs/quickstart.rst

```diff
@@ -79,6 +79,28 @@ Example:
 The raw values (not-casted to any type) are still available from their
 usual place on the request (`matchdict`, `GET`, `POST`, `json()`, etc)
 
+If you have ``pyramid_swagger.use_models`` enabled, you can interact with
+models defined in ``#/definitions`` as Python classes instead of dicts.
+
+.. code-block:: python
```

Comment:
Can we include the snippet of the Swagger model here too? I think those three together will make it very clear what's going on.